src/common/peripheral.ts

```diff
@@ -0,0 +1,12 @@
+/********************************************************************************
+ * Copyright (C) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License as outlined in the LICENSE File
+ ********************************************************************************/
+
+/**
+ * Those peripheral names will be ignored in the peripheral tree.
+ * They will not be resolved from the SVD file.
+ */
+export const IGNORE_PERIPHERAL_NAMES = ['AXBS0', 'CAU3'];
```

Comment:
I start liking the idea of having an ignore list. But we probably shouldn't hard-code it. Rather make it configurable.

Reply:
How would you like this to be configurable? A VS Code setting or should that be something on the API level?